src/util/userHelpers.js

```diff
@@ -167,3 +167,20 @@ export const isUserAuthorized = (currentUser, permissionsToCheck) => {
     ? isActive && hasPermissionToPostListings(currentUser)
     : isActive;
 };
+
+/**
+ * Check if currentUser has permission to post listings.
+ *
+ * @param {Object} currentUser API entity
+ * @returns {Boolean} true if currentUser has permission to post listings.
+ */
+export const hasPermissionToInitiateTransactions = currentUser => {
```

Comment:
This logic assumes that once the Core side is done, all users get `initiateTransactions: 'permission/allow'` in their effectivePermissionSet unless explicitly prohibited in the UI. 

Is that a correct assumption, or should this check the accessControl.json asset to see whether the marketplace restricts initiating transactions or not?

Reply:
Function moved!

As far as the permission check, I don't have a super strong opinion. My intuition is that when we have a larger array of these different permissions, then customizers may want to create different kinds of behaviors for different use cases, in which case it would be useful for them if each function only does a single thing and not a big isUserAuthorized function that wraps all that behavior out of sight. It might also help folks understand what exactly is being allowed and restricted when they read the code.

Having atomic functinos would also guide the devs into making explicit decisions on what permissions go in what features and not rely on the isUserAuthorized function to make the decisions for them. 

If we decide to wrap things into the isUserAuthorized function, then folks can of course modify the logic there, too, so either way works for me.

Reply:
After an IRL conversation we ended up simplifying the isUserAuthorized helper function to just check currentUser state and not accept permissionsToCheck param, so those changes have been implemented!